services/payload/src/components/SideNav/SideNav.scss

```diff
@@ -0,0 +1,78 @@
+.navbar-wrapper {
+    background-color: #FFFFFF;
+    box-shadow: 4px 0px 4px 0px rgba(0, 0, 0, 0.10);
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    align-items: center;
+    position: fixed;
+    z-index: 100;
+    width: 240px;
+}
+
+.navbar-icon-wrapper {
+    margin-top: 60px 
+}
+
+.navbar-buttons-wrap {
+    display: flex;
+    flex-direction: column;
+    height: 215px;
+    margin-top: 90px;
+}
+
+.navbar-buttons {
+    background-color: #FFFFFF;
+    border: none;
+    border-radius: 100px;
+    margin-bottom: 10px;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    font-size: 16px;
+    text-decoration: none;
+
+    & > img {
+        width: 22px;
+        height: 22px;
+        margin-right: 10px;
+    }
+}
+
+.active {
```

Comment:
This will accidentally match all kinds of stuff it shouldn't, and so should get nested to only match if an element has _both_ `navbar-buttons` and `active` classes!